general/project/projectinit.go

```diff
@@ -211,20 +225,15 @@ func createProjectBuildConfigs(tech coreutils.Technology, projectPath string, se
 		configFile.Resolver.SnapshotRepo = MavenVirtualDefaultName
 		configFile.Deployer.ReleaseRepo = MavenVirtualDefaultName
 		configFile.Deployer.SnapshotRepo = MavenVirtualDefaultName
-	case coreutils.Gradle:
-		configFile.Resolver.Repo = GradleVirtualDefaultName
-		configFile.Deployer.Repo = GradleVirtualDefaultName
-	case coreutils.Npm:
-		configFile.Resolver.Repo = NpmVirtualDefaultName
-		configFile.Deployer.Repo = NpmVirtualDefaultName
-	case coreutils.Go:
-		configFile.Resolver.Repo = GoVirtualDefaultName
-		configFile.Deployer.Repo = GoVirtualDefaultName
-	case coreutils.Pipenv:
+	case coreutils.Dotnet:
 		fallthrough
-	case coreutils.Pip:
-		configFile.Resolver.Repo = PypiVirtualDefaultName
-		configFile.Deployer.Repo = PypiVirtualDefaultName
```

Comment:
Go and Pip seem to have been removed from here. Any reason why?

Reply:
As well as Gradle and npm. I thought it look nicer to deal with the general (default) case using the map we already have (RepoDefaultName). @eyalbe4 do you think otherwise?